xml/System.Text.Json.Serialization/ReferenceHandling.xml

```diff
@@ -34,9 +34,22 @@
         <ReturnType>System.Text.Json.Serialization.ReferenceHandling</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Serialization does not support objects with cycles and does not preserve duplicate references. Metadata properties will not be written when serializing reference types and will be treated as regular properties on deserialize.</summary>
         <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+* On Serialize:
+ Treats duplicate object references as if they were unique and writes all their properties.
+ The serializer throws a <xref:System.Text.Json.JsonException> if an object contains a cycle.
+ * On Deserialize:
+ Metadata properties (`$id`, `$values`, and `$ref`) will not be consumed and therefore will be treated as regular JSON properties.
+ The metadata properties can map to a real property on the returned object if the property names match, or will be added to the <xref:System.Text.Json.Serialization.JsonExtensionDataAttribute> overflow dictionary, if one exists; otherwise, they are ignored.
```

Comment:
```suggestion
 The metadata properties can map to a real property on the returned object if the property names match. Or, if the <xref:System.Text.Json.Serialization.JsonExtensionDataAttribute> overflow dictionary exists, the metadata properties are added to it. Otherwise, the metadata properties are ignored.
```